packages/autorest.typescript/src/restLevelClient/generateRestLevel.ts

```diff
@@ -135,7 +135,7 @@ export async function generateRestLevelClient() {
     const isJson = /\.json$/gi.test(filePath);
     const isSourceCode = /\.(ts|js)$/gi.test(filePath);
     let fileContents = fs.readFileSync(filePath);
-    const licenseHeader = `// Copyright (c) Microsoft Corporation.\n// Licensed under the MIT license.\n`;
+    const licenseHeader = `// Copyright (c) Microsoft Corporation.\n// Licensed under the MIT License.\n`;
```

Comment:
codegen was using lower case so I am not 100% sure why some test baselines have upper case and some have lower case.